src/hooks/useOnboardingStatus.ts

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from 'react';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { ONBOARDING_STATUS_KEY } from '@src/constants/storage';
+
+export const useOnboardingStatus = () => {
+	const [hasOnboardingSeen, setHasOnboardingSeen] = useState(false);
+
+	const loadStatus = async () => {
+		const value = await AsyncStorage.getItem(ONBOARDING_STATUS_KEY);
+		setHasOnboardingSeen(value === 'true');
+	};
```

Comment:
You can declare this function inside useEffect hook. This way you wont need to add `biome-ignore`